apps/web-mzima-client/src/app/settings/data-export/data-export.component.html

```diff
@@ -13,7 +13,8 @@ <h1>{{ (hxlEnabled ? 'data_export.title_hxl' : 'data_export.title') | translate
           <p>{{ 'data_export.hxl_apikey_alert_1' | translate }}</p>
           <p>
             {{ 'data_export.hxl_apikey_alert_2' | translate }}
-            <a [routerLink]="['/settings/user-settings']" class="link-blue">
+            <!--Inadequate Color Contrast for Accessibility -->
+            <a [routerLink]="['/settings/user-settings']" class="colorLink">
               {{ 'data_export.hxl_configure' | translate }}
```

Comment:
A question here, what happens if your entirely remove the class?

Reply:
Hello there  Yes, the class "link-blue" is used elsewhere, and when I tried to add a new class name, the original class property was overriding the new class property. Therefore, I removed the original class and set the text color to black for the new class. and removing original the class is not affect any othere  properties.